backends/s3/s3.go

```diff
@@ -108,6 +117,9 @@ func (o Options) Check() error {
 	if !hasSecretsCreds && !hasStaticCreds {
 		return fmt.Errorf("s3 storage.options: credentials are required, fill either (access_key and secret_key) or (access_key_filename and secret_key_filename)")
 	}
+	if d := o.SecretsRefreshInterval; hasSecretsCreds && d != 0 && d < time.Second {
+		return fmt.Errorf("s3 storage.options: field refresh_secrets is required when using secret credentials")
```

Comment:
This is not required, because we fall back to the default. 
The error message is misleading when using a too low value. 
The field name in the message is incorrect.

Instead:

```suggestion
		return fmt.Errorf("s3 storage.options: field secrets_refresh_interval must be at least 1s")
```